validation-test/ParseableInterface/verify_stdlib.swift

```diff
@@ -0,0 +1,7 @@
+// Note that this test should still "pass" when Swift.swiftinterface has not
+// been generated.
+
+// RUN: %empty-directory(%t)
+// RUN: test ! -e %platform-module-dir/Swift.swiftmodule/%target-cpu.swiftinterface || %target-swift-frontend -build-module-from-parseable-interface %platform-module-dir/Swift.swiftmodule/%target-cpu.swiftinterface -parse-stdlib -o %t/Swift.swiftmodule
```

Comment:
@compnerd Is this good on Windows?

Reply:
Oops, right. I copied from `verify_simd` but that's guarded by `objc_interop` anyway.